### DISCUSSION TITLE
Maiana - "permission denied, cannot connect to device"

### DISCUSSION THREAD

**Discussion by AlexBigwood** [peterantypas/maiana#116] [Q&A]
Hello  - Wondering if anyone can point me in the right direction here?

Have installed Maiana as per directions (via USB), app will not connect to the device and remains in a constant state of trying to read device settings. 

Notes:
When I built the antenna, I did it together with a friend, and tested it on his Pi first.

In the setup directions, it recommends 'remember device' by over 'remember port,' however It will not give me the option and automatically opts to 'remember port.' Not sure if this makes much difference?

Signal K error reads as "permission denied, cannot connect to /devttyOP_maiana"

have reinstalled several times, updated all apps etc. No luck.

I also posted in Open Marine - Not sure which is the best forum for this - Apologies

Pi 3B+
Maiana ais app v2.1.2
Signal K v1.44.0
Serial v2.2.3
![Untitled](https://user-images.githubusercontent.com/112564592/187700399-72322f8f-8a88-4720-b3df-129335782ada.png)

**Comment by sailoog** [comment]
that seems an openplotter issue, better go here: https://forum.openmarine.net/showthread.php?tid=4208
